CMakeLists.txt

```diff
@@ -38,6 +38,15 @@ if( IBIS_USE_QT_MULTIMEDIA )
     set( IbisQtModules ${IbisQtModules} Multimedia )
 endif()
 find_package( Qt5 COMPONENTS ${IbisQtModules} REQUIRED )
+
+if (Qt5Widgets_FOUND)
+    if (Qt5Widgets_VERSION VERSION_LESS 5.9.0)
+        message(FATAL_ERROR "Minimum supported Qt5 version is 5.9.0!")
```

Comment:
Maybe in the case of Qt we could simply have a warning: "Ibis officially supports Qt version > 5.9 and you are using xxx. It might run anyways.". At the moment, we don't use any feature that requires Qt > 5.2 so we can let people some time to get used to the idea of installing a fixed version of Qt.